runtime/rococo/src/lib.rs

```diff
@@ -627,16 +627,16 @@ pub type XcmRouter = (
 	xcm_sender::ChildParachainRouter<Runtime>,
 );
 
-use xcm::v0::{MultiAsset, MultiAsset::AllConcreteFungible, MultiLocation::{Null, X1}, Junction::Parachain};
+use xcm::v0::{MultiAsset, MultiAsset::AllConcreteFungible, Junctions::{Null, X1}, Junction::Parachain};
 parameter_types! {
 	pub const RococoForTick: (MultiAsset, MultiLocation) =
-		(AllConcreteFungible { id: Null }, X1(Parachain(100)));
+		(AllConcreteFungible { id: MultiLocation::empty() }, MultiLocation::with_parachain_interior(100));
```

Comment:
```suggestion
		(AllConcreteFungible { id: MultiLocation::empty() }, Parachain(100).into());
```
Should also work, right?

Reply:
Yes, however the proper syntax is:

```suggestion
		(AllConcreteFungible { id: MultiLocation::empty() }, X1(Parachain(100)).into());
```

I actually opted to use `with_parachain_interior` because it's clearer on what the code is doing.